app/llm_integration.py

```
@@ -0,0 +1,740 @@
+                    final_context = final_context.split("```json", 1)[1]
+                if "```" in final_context:
+                    parts = final_context.split("```")
+                    for part in parts:
+                        if part.strip() and not part.strip().startswith('{') and not part.strip().startswith('['):
+                            continue
+                        final_context = part.strip()
+                        break
+                
+                # Validate JSON
+                json.loads(final_context)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)